pkg/cloudprovider/providers/azure/azure_blobDiskController.go

```diff
@@ -481,7 +469,7 @@ func (c *BlobDiskController) getAllStorageAccounts() (map[string]*storageAccount
 
 	accounts := make(map[string]*storageAccountState)
 	for _, v := range *accountListResult.Value {
-		if strings.Index(*v.Name, storageAccountNameMatch) != 0 {
+		if strings.Index(*v.Name, azureDiskAccountNamePrefix) != 0 {
```

Comment:
The code accesses *v.Name first, and check v.Name == nil in next block? Shall fix that?

Reply:
Good catch. fixed.